app/tests/database.py

```diff
@@ -104,21 +128,42 @@ def create_item_links(item):
     return [link]
 
 
-def create_asset(item, name):
-    asset = Asset.objects.create(
+@override_settings(
+    AWS_STORAGE_BUCKET_NAME='mybucket',
+    AWS_ACCESS_KEY_ID='mykey',
+    AWS_DEFAULT_ACL='public-read',
+    AWS_S3_REGION_NAME='wonderland',
+    AWS_S3_ENDPOINT_URL=None
+)
+@mock_s3
+def create_asset(item, name, eo_gsd=3.4, geoadmin_variant="kgrs", proj_epsg=2056):
+    # Check if the bucket exists and if not, create it
+    s3 = boto3.resource('s3', region_name='wonderland')
+    try:
+        s3.meta.client.head_bucket(Bucket='mybucket')
+    except botocore.exceptions.ClientError as e:  # pylint: disable=invalid-name
```

Comment:
I would use `error` instead of `e` here, then the `pylint: disable=invalid-name` is not needed

Reply:
changed